direct/src/gui/OnscreenGeom.py

```diff
@@ -111,6 +113,7 @@ def setGeom(self, geom,
                 self.setColor(color[0], color[1], color[2], color[3])
 
     def getGeom(self):
+        warnings.warn("stop using this function",FutureWarning)
```

Comment:
Style-wise, there should be a space after the comma.  We could probably improve the wording of the warning message, like "This method is vestigial and will be removed shortly."